docs/contributors/repository-management.md

```diff
@@ -70,6 +70,14 @@ Gutenberg follows a feature branch pull request workflow for all code and docume
 2. Make your changes, testing thoroughly.
 3. Commit your changes when you’re happy with them, and push the branch.
 4. Open your pull request.
+5. Label and name your pull request appropriately (see below).
```

Comment:
This likely needs to be adjusted to the following since non contributors won't have the ability to self label their own PRs: 

5. If you have the ability to do so, label and name your pull request appropriately (see below).